js/doc.js

```diff
@@ -113,13 +114,18 @@ function SchemaPart({ key, property, parent, parentSlug }) {
         type = `[]object`;
       }
     }
+    if (property.Enum) {
+      type = type + ': enum';
```

Comment:
We should probably use template strings:

```suggestion
      type = `${type}: enum`
```